tests/yast2_gui/yast2_keyboard.pm

```diff
@@ -45,7 +45,7 @@ sub run {
     send_key "g";
 
     # 2. Switch keymap from us to german
-    send_key_until_needlematch("yast2_keyboard-layout-german", "down");
+    send_key_until_needlematch("yast2_keyboard-layout-german", "down") unless check_screen("yast2_keyboard-layout-german", 30);
```

Comment:
I m curious, could we avoid having this `unless` (which can be confusing to someone who is not aware of the problem), by just making the `yast2_keyboard-layout-german` needle wider to include the empty space on the right?
It will then only match the plain german version and the code will stay as it was.

Reply:
it is worse than that, we send `alt-g` before that, so we are supposed to be on the item, and then we always iterate +1 for what the test is designed :(
so to now overthink much I think it is acceptable. sometimes the 'alt-g' doesn't reach the SUT and there is not problem, it starts from item 1 to the german item we want.